### PR TITLE
Renaming tile entities to block entities

### PR DESCRIPTION
**Pull request by Dweblenod**
Using the refactoring tool in IntelliJ to rename tile entities to match 1.18's naming scheme

### CODE REVIEW

File: src/main/java/com/mraof/minestuck/computer/ComputerReference.java [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
 public interface ComputerReference
 {
-	static ComputerReference of(ComputerTileEntity te)
+	static ComputerReference of(ComputerBlockEntity te)
```
Comment (Dweblenod) [inline src/main/java/com/mraof/minestuck/computer/ComputerReference.java:12]: on line 20 below, is it appropriate to change tile_entity to block_entity if I also change it in TEComputerReference? Also may TEComputerReference be renamed to BEComputerReference?

Comment (kirderf1) [reply src/main/java/com/mraof/minestuck/computer/ComputerReference.java]: Yes and yes

Comment (Dweblenod) [reply src/main/java/com/mraof/minestuck/computer/ComputerReference.java]: And to double confirm, I do not need to change that string(tile_entity) in any other file to the best of your knowledge?

Comment (kirderf1) [reply src/main/java/com/mraof/minestuck/computer/ComputerReference.java]: Indeed, that nbt key should only be used in those two places.

Comment (Dweblenod) [reply src/main/java/com/mraof/minestuck/computer/ComputerReference.java]: If thats the case then there are no other issues I can detect with this PR

**Review by kirderf1 — APPROVED** [review]
I don't think there's much reason to review this one thoroughly, so here's an approval.

**Pull request merged**